src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -433,6 +433,11 @@ private void ImportCall(ILOpcode opcode, int token)
                                 _dependencies.Add(_factory.ReadyToRunHelper(ReadyToRunHelperId.DelegateCtor, info), reason);
                             }
 
+                            // TODO-LLVM: delete when IL->LLVM module is gone.
+                            if (_compilation.TargetArchIsWasm())
+                            {
+                                _dependencies.Add(info.Constructor, "LLVM delegate ctor");
+                            }
```

Comment:
There were some scenarios where scanning missed the appropriate intrinsic initializer; I am pretty sure this is overly conservative but debugging deeper did not seem worth it.